Experimental/src/org/sleuthkit/autopsy/experimental/textclassifier/TextClassifierTrainerConfigPanel.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.experimental.textclassifier;
+
+import javax.swing.JPanel;
+
+class TextClassifierTrainerConfigPanel extends JPanel {
```

Comment:
I  believe this class is unnecessary.

Reply:
Is this what I should put in TextClassiferTrainer:
`private JPanel configPanel = new JPanel();`

Reply:
I think either returning null or using the default implementation (which returns null) of GeneralReportModuleAdapter if you decide to extend it instead of implementing the same interface as it would be the way we typically indicate there is no panel for these.